modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -688,6 +688,41 @@ AUI.add(
 					instances[id] = inputLocalizedInstance;
 				}
 
+				if (config.editor) {
+					window.CKEDITOR.on('editorChanged', (event) => {
```

Comment:
It doesn't feel safe to access `window.CKEDITOR` like this. Based on what we know, it's unlikely this can happen, but nothing ensures `CKEDITOR` is defined here. If we protected against that, it would mean that the bug is still present under those circumstances

Reply:
@julien, does `config.editor` change at all? I'd expect that API to be stable and maybe something we could work with instead of a global thing such as `window.CKEDITOR`.

Reply:
@jbalsas I'll have to check to make sure, but what I remember is that it doesn't change.

I agree with you that using `config.editor` would be much cleaner and safer than using the global `window.CKEDITOR` but I when I tried to use that, I remember facing a problem; I will dive in again to see what I can do about it and comment with more details.

Reply:
The other thing is that the `input-editor` tag already takes an `onInit` method. Bad thing is that you have to declare a global function to pass it down, but since it's already an existing API, this might be the shortest path to your goal. 

```
<c:when test='<%= type.equals("editor") %>'>
	<liferay-ui:input-editor
		contents="<%= mainLanguageValue %>"
		contentsLanguageId="<%= languageId %>"
		cssClass='<%= "language-value " + cssClass %>'
		editorName="<%= editorName %>"
		name="<%= inputEditorName %>"
		onChangeMethod='<%= randomNamespace + "onInitEditor" %>'
		onChangeMethod='<%= randomNamespace + "onChangeEditor" %>'
		placeholder="<%= placeholder %>"
		toolbarSet="<%= toolbarSet %>"
	/>

	<aui:script>
		function <%= namespace + randomNamespace %>onChangeEditor() {
                    ...
		}

		function <%= namespace + randomNamespace %>onInitEditor() {
			var inputLocalized = Liferay.component('<%= namespace + HtmlUtil.escapeJS(fieldName) %>');

			var editor = window['<%= namespace + HtmlUtil.escapeJS(inputEditorName) %>'];

                        // This might be a new API if needed
			inputLocalized.doWhatever(editor);
		}
	</aui:script>
</c:when>

Reply:
@jbalsas I've updated everything in [https://github.com/liferay-frontend/liferay-portal/pull/939/commits/cf30f36c49a5c565dfe446c06ce5a626990f4c85](https://github.com/liferay-frontend/liferay-portal/pull/939/commits/cf30f36c49a5c565dfe446c06ce5a626990f4c85).

Instead of dispatching an event from `ckeditor.jsp`, the approach I took is the following:
- Subscribe to the editor's `destroy` event
- Periodically check that the editor's status is `ready`
- Update the `InputLocalized`'s `inputPlaceholder` property with the new value.

The reason why this "workflow" seems so complex is that when the viewport is resized, the editor is destroyed to update it's toolbars and so the `editor` reference we have in `input_localized` is no longer valid (which means we won't get notified when it dispatches events). I went this way to avoid using globals.

Reply:
Did you try the approach I outlined above? I'm not sure polling the editors for status is better than using the global `CKEDITOR`...

Reply:
Sorry, when I added my last comment, for some reason I didn't see yours.

I was about to add a similar comment about polling which I think is a very bad idea.
I'm going to try using `onInit` like you commented